CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.34.2
+
+*January 12, 2021*
+
+This release fixes a substantial bug in evidence handling where evidence could
+sometimes be broadcast before the block containing that evidence was fully committed,
```

Comment:
```suggestion
sometimes be broadcasted before the block containing that evidence was fully committed,
```

?

Reply:
Lol we just had this discussion because I pointed out the same thing. Apparently broadcast is the same word in present and past tense.
```
("Broadcasted" is not standard English.)
```

Reply:
wasn't sure about it either, thanks

Reply:
I wasn't either honestly, lol